0067-quarterly-releases.md

```diff
@@ -0,0 +1,53 @@
+- Feature Name: Release Schedule
+- Start Date: 2022-04-21
+- RFC PR: [apache/tvm-rfcs#67](https://github.com/apache/tvm-rfcs/pull/67)
+
+# Summary
+This RFC proposes that TVM move to a quarterly release schedule. Releases would happen every 3 months or so on a schedule set well in advance, independent of individual feature development in TVM.
+
+# Motivation
+Releases are essential to the usage of TVM, especially now that are beginning to work on publishing binary packages for TVM under PyPi. Making TVM releases frequent forces the release process to become well documented and simple rather than bespoke and only achievable by a small group. Users benefit from releases by seeing that the the project is still under active development and providing an easy way to get new features. As of this RFC it has been five months since the last release. It could easily confuse new users when they expect some TVM feature that was only developed recently but is not present in the latest official release.
+
+
+# Guide-level explanation
+TVM has [release process documentation](https://tvm.apache.org/docs/contribute/release_process.html). This RFC proposes that the release candidate vote thread be abolished in favor of a mechanical schedule where releases happen roughly every three months. A release branch will be cut, evaluated for a period of two weeks, then a release published. Publishing a release entails:
```

Comment:
Doesn't this RFC just provide a schedule for following the release process? The release candidate vote thread is part of that process and ensures we don't mistakenly ship a broken release, why do we want to abolish it as part of this rather than incorporate it into the two week review window?

Reply:
Thanks @driazati for proposing the change and bringing up the same goal of making release streamlined (which we all share).

Just to clarify this particular point(of release voting) as it is something that is related to the policy. Overall release policy is something that apache handles (sometimes extra) carefully overall. Release voting thread is mandatory per  policy and it is not something that we can change project-wide. 

The rationale is that the release voting thread brings broad community awareness and overall license requirements.  A release needs at least three PMC vote to be shipped. We can however, keep the schedule streamlined and improve the overall efficiency driven by a release manager.

The original release policy guide is a good reference pt -- about the GPG key needs and where to upload, posts.

Reply:
Ah, I wasn't aware of the rules on the vote thread. I think it is fine to keep in that case but move it to the end of the process, so the release manager does most of the work and testing but sends it to the PMC for approval before publishing anything

Reply:
no need to explicit send to PMC as PMC members normally vote on the same public voting thread